src/GoogleCloudPubSubAdapter.php

```diff
@@ -207,10 +208,12 @@ public function subscribe($channel, callable $handler)
                 if ($payload === 'unsubscribe') {
                     $isSubscriptionLoopActive = false;
                 } else {
-                    call_user_func($handler, $payload);
+                    $response = call_user_func($handler, $payload);
+                }
+                
+                if (!$checkResponse || isset($response) && $response) {
```

Comment:
just for some compatibility.. what if we said `if($reponse === false) {`?

Reply:
Compatibility? This is backwards compatible by default.
It retains the default expected behaviour by setting `$checkResponse` to false.
Only if it is set to true do we care about the response from the callback.
And then we also check that a response was returned and that that value is true or truthy.
```php
>>> $checkResponse = false;
=> false
>>> (!$checkResponse || isset($response) && $response)
=> true
>>> $checkResponse = true;
=> true
>>> (!$checkResponse || isset($response) && $response)
=> false
>>> $response = true;
=> true
>>> (!$checkResponse || isset($response) && $response)
=> true
```